app/components/course_grid_component/course_grid_component.html.erb

```diff
@@ -49,10 +49,10 @@
           </div>
       </div>
     </div>
-    <p class="govuk-body govuk-!-text-align-centre">
+    <p class="govuk-body govuk-!-text-align-centre govuk-!-margin-bottom-0">
```

Comment:
Spacing was a bit too much at the bottom of the course component - this is to reduce it slightly and look neater.